recipes/mysql-concpp/all/test_package/conanfile.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+from conan import ConanFile
+from conan.tools.cmake import CMake, cmake_layout
+from conan.tools.build import can_run
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "CMakeDeps", "CMakeToolchain"
+
+    def requirements(self):
+        self.requires(self.tested_reference_str)
+
+    def build_requirements(self):
+        self.tool_requires("cmake/[>=3.15 <4]")
+
```

Comment:
```suggestion
```
Can be avoided since Conan v2 assumes and requires that CMake 3.15+ is present on the system.